### PR TITLE
Master review

### PR DESCRIPTION
**Pull request by duardoqueiroz**
# Proposed Changes
- User profile card
- Master review system

# Screenshots

> ## Update
image
> #
image
> #
image

### CODE REVIEW

**Review by leticiabsduarte — APPROVED** [review]
It looks perfect, good work =)

File: resources/views/components/review-edit.edge [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<form action="/{{element}}/{{id}}/reviews?_method=PUT" method="POST">
+  <div x-data="{
+    content: '',
+    limit: $el.dataset.limit,
+    get remaining() {
+      return this.limit - this.content.length
+    } }" data-limit="300" ,
```
Comment (Manuel-Antunes) [inline resources/views/components/review-edit.edge:7]: Please, move this state to app.js

File: resources/views/components/review-form.edge [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    padding-left:0;
+    padding-right:.2rem;
+    margin: 0;
+    cursor: pointer;
+}
+.rate .fa-star-half-alt:before {
+    position: absolute;
+    padding-right: 0;
+}
+</style>
```
Comment (Manuel-Antunes) [inline resources/views/components/review-form.edge:16]: Try to config this styles in head

File: resources/views/components/star-rating.edge [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div x-data="{ temp: 0, orig: 0 }" class="flex cursor-pointer" @click="orig = temp">
```
Comment (Manuel-Antunes) [inline resources/views/components/star-rating.edge:1]: Call this state from app.js

File: resources/views/components/star-value.edge [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<div x-data="{
+  starLimit:5,
+  rating: {{rating}},
+}" class="flex flex-row">
```
Comment (Manuel-Antunes) [inline resources/views/components/star-value.edge:4]: Call this state from app.js

File: resources/views/users/index.edge [hunk 5]
```diff
@@ -1,21 +1,248 @@
...
-  <div class="left-20 bottom-10 absolute flex items-center text-lg font-semibold text-white">
-    <img class="w-20 h-20 object-cover rounded-full" src="{{user.photoUrl}}" alt="{{user.name}}">
-    <span class="ml-10">Name: {{user.name}}</span>
-  </div>
-</div>
-  <div class="container mx-auto flex flex-col">
-    <div class="flex">
-      <span>Name: {{user.name}}</span>
-      <span>Email: {{user.email}}</span>
+<section class="bg-gray-900 h-full" x-data="{contentId:1}">
```
Comment (Manuel-Antunes) [inline resources/views/users/index.edge:8]: call this state from app.js

**Review by Manuel-Antunes — CHANGES_REQUESTED** [review]
Check my comments, solve, and request my review again

**Review by leticiabsduarte — APPROVED** [review]
I got lost looking for where to review the master, but it keeps perfect to me. Good work!

**Pull request closed without merging**